src/main/java/com/conveyal/datatools/manager/controllers/api/DeploymentController.java

```diff
@@ -264,7 +277,7 @@ private static Deployment createDeploymentFromFeedSource (Request req, Response
      * Update a single deployment. If the deployment's feed versions are updated, checks to ensure that each
      * version exists and is a part of the same parent project are performed before updating.
      */
-    private static Deployment updateDeployment (Request req, Response res) {
+    private static Deployment updateDeployment (Request req, Response res) throws CheckedAWSException {
```

Comment:
This exception no longer gets thrown.
```suggestion
    private static Deployment updateDeployment (Request req, Response res) {
```

Reply:
Removed in 1f5673d390acd5a8af7ad68b32260b14e5afe526